server/data/eqSearchApiClient.ts

```diff
@@ -0,0 +1,33 @@
+import RestClient from './restClient'
+
+type searchResult = {
```

Comment:
We can rename this type to **Result**

Reply:
Seeing as everything else here is prefixed with `search` or `eqSearch` I think `Result` doesn't follow. Id disagree and say the current type name is fine. Although we could then argue that line 17 the response interface should be `searchResults: searchResult[]` 🙂  So I'm happy as long as everything is consistent.

Reply:
Thanks for your reviews @muyinatech @psoleckimoj, I think I will then keep it as it is as I found it it less confusing, but happy to come back on it in the future to make it consistent with the whole project if necessary :)